### DISCUSSION TITLE
Diesel should support u64 for sqlite

### DISCUSSION THREAD

**Discussion by s4093120** [diesel-rs/diesel#3469] [Ideas]
![image](https://user-images.githubusercontent.com/62601155/211498416-d7efd8c0-a86b-4be5-a107-86033e019435.png)
I was scrolling through the docs and noticed UNIGNED BIT INT existed.

https://www.sqlite.org/c3ref/int64.html
for the c api it seems it supports it as well

**Comment by weiznich** [comment]
Your image is part of the [type affinity example table from the SQLite documentation](https://www.sqlite.org/datatype3.html). The first column shows type name strings and the second column shows the underlying type affinity used by SQLite. In turn that means even if you specify the type as `UNSIGNED BIG INT` SQLite will use a `INTEGER` column as storage. This column can hold data between `i64::MIN` and `i64::MAX`, which is different than having a `u64` there.
